From 819b28907a4a95e9711782162c65fc946ecd2dc6 Mon Sep 17 00:00:00 2001
Message-ID: <819b28907a4a95e9711782162c65fc946ecd2dc6.1705743628.git.sharlatanus@gmail.com>
In-Reply-To: <cover.1705743627.git.sharlatanus@gmail.com>
References: <cover.1705743627.git.sharlatanus@gmail.com>
From: Sharlatan Hellseher <sharlatanus@gmail.com>
Date: Sat, 20 Jan 2024 09:38:44 +0000
Subject: [PATCH 38/38] gnu: go-github-com-quic-go-qtls-go1-20: Sort package
 alphabetically.

* gnu/packages/golang-crypto.scm (go-github-com-quic-go-qtls-go1-20):
Sort package alphabeticaly to consist the style.

Change-Id: Ib9d1e037dc1172b33a80dc4b40c785b53a2a0586
---
 gnu/packages/golang-crypto.scm | 54 +++++++++++++++++-----------------
 1 file changed, 27 insertions(+), 27 deletions(-)

diff --git a/gnu/packages/golang-crypto.scm b/gnu/packages/golang-crypto.scm
index b4c3ecdecd..80e87e3f00 100644
--- a/gnu/packages/golang-crypto.scm
+++ b/gnu/packages/golang-crypto.scm
@@ -655,33 +655,6 @@ (define-public go-github-com-multiformats-go-multihash
       (description "Multihash implementation in Go.")
       (license license:expat))))
 
-(define-public go-github-com-quic-go-qtls-go1-20
-  (package
-    (name "go-github-com-quic-go-qtls-go1-20")
-    (version "0.3.4")
-    (source
-     (origin
-       (method git-fetch)
-       (uri (git-reference
-             (url "https://github.com/quic-go/qtls-go1-20")
-             (commit (string-append "v" version))))
-       (file-name (git-file-name name version))
-       (sha256
-        (base32 "0fl3yv1w8cygag3lav45vvzb4k9i72p92x13wcq0xn13wxirzirn"))))
-    (build-system go-build-system)
-    (arguments
-     (list
-      #:import-path "github.com/quic-go/qtls-go1-20"
-      #:go go-1.20))
-    (propagated-inputs
-     (list go-golang-org-x-crypto
-           go-golang-org-x-sys))
-    (home-page "https://github.com/quic-go/qtls-go1-20")
-    (synopsis "TLS 1.3 for QUIC")
-    (description "Go standard library TLS 1.3 implementation, modified for
-QUIC.  For Go 1.20.")
-    (license license:expat)))
-
 (define-public go-github-com-operatorfoundation-ed25519
   (let ((commit "b22b4bd3ddef042eec45f3ee135cd40281fde2b4")
         (revision "0"))
@@ -744,6 +717,33 @@ (define-public go-github-com-protonmail-go-crypto
 official package.")
     (license license:bsd-3)))
 
+(define-public go-github-com-quic-go-qtls-go1-20
+  (package
+    (name "go-github-com-quic-go-qtls-go1-20")
+    (version "0.3.4")
+    (source
+     (origin
+       (method git-fetch)
+       (uri (git-reference
+             (url "https://github.com/quic-go/qtls-go1-20")
+             (commit (string-append "v" version))))
+       (file-name (git-file-name name version))
+       (sha256
+        (base32 "0fl3yv1w8cygag3lav45vvzb4k9i72p92x13wcq0xn13wxirzirn"))))
+    (build-system go-build-system)
+    (arguments
+     (list
+      #:import-path "github.com/quic-go/qtls-go1-20"
+      #:go go-1.20))
+    (propagated-inputs
+     (list go-golang-org-x-crypto
+           go-golang-org-x-sys))
+    (home-page "https://github.com/quic-go/qtls-go1-20")
+    (synopsis "TLS 1.3 for QUIC")
+    (description "Go standard library TLS 1.3 implementation, modified for
+QUIC.  For Go 1.20.")
+    (license license:expat)))
+
 (define-public go-github-com-refraction-networking-utls
   (package
     (name "go-github-com-refraction-networking-utls")
-- 
2.41.0

